src/cmd/package.go

```diff
@@ -300,6 +302,24 @@ func identifyAndFallbackToClusterSource() (src sources.PackageSource) {
 	return src
 }
 
+func getPackageCompletionArgs(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
```

Comment:
```suggestion
func getPackageCompletionArgs(_ *cobra.Command, _ []string, _ string) ([]string, cobra.ShellCompDirective) {
```
Some lints flagged on this function (unused vars can be underscored.